resources/monitoring/templates/kyma-additions/alertmanager.config.yaml

```diff
@@ -47,7 +47,7 @@ receivers:
 - name: "victorOps"
   victorops_configs:
   - api_key: {{ .Values.global.alertTools.credentials.victorOps.apikey | quote }}
-    send_resolved: true
+    send_resolved: {{ .Values.global.alertTools.credentials.victorOps.sendResolved | quote }}
```

Comment:
```suggestion
    send_resolved: {{ .Values.global.alertTools.credentials.victorOps.sendResolved }}
```

Reply:
hi @rakesh-garimella , thanks for your suggestion and now i followed your suggestion. to be honest, after its succeed upgrading, some test-suites are failed (e.g.               {"code":404,"error":"Deleting service from Application repository failed, Application appk8srestest1-g7jxx not found"}
) ....